templates/docs/examples/layouts/docs.html

```diff
@@ -163,7 +163,7 @@ <h2 class="p-table-of-contents__header">On this page</h2>
     </div>
   </div>
 
-  <main class="l-docs__main">
+  <main class="l-docs__main" id="main-content">
```

Comment:
I understand the intention here, but this way jumping to "main-content" would skip the title, which is earlier in "l-docs__title" part. I believe it's more useful when skipping to main content to actually start with the title and whatever is below it.

Yes, this layout makes it a bit problematic as it splits the "main" area into couple blocks (title and main) with table of contents in between them. But I'm not sure if there is another good way around this.

Reply:
Ah ok, in that case I will close this.